ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
+  "filename": "whitestorm.js",
+  "version": "1.2.6",
+  "homepage": "https://whsjs.readme.io/docs",
+  "author": "sasha240100 <siteprogcom@gmail.com>",
```

Comment:
I think it's better to be the same as upstream. Ref:https://github.com/WhitestormJS/whitestorm.js/blob/beta/package.json#L27